src/Sentiment/loader.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This is an extra file perhaps that shouldn't be here?

Reply:
Yes. we don't need this.